getting-started/vs-integration/introduction.md

```diff
@@ -10,18 +10,20 @@ position: 1
 
 The Progress&reg; Telerik&reg; UI for Blazor Visual Studio (VS) Extensions enhance the experience in developing Blazor web applications with Telerik UI for Blazor.
 
->caption The Telerik Blazor Extensions provide the following advantages:
```

Comment:
Captions are not required to introduce lists.

Reply:
There are some rogue captions due to the CTA panels insertions where we tried to use them to contrl wher the CTA panel is.